etna/media/forms.py

```diff
@@ -0,0 +1,64 @@
+import magic
+
+from django import forms
+from django.core.exceptions import ValidationError
+
+from wagtail.admin import widgets
+
+from wagtailmedia.permissions import permission_policy as media_permission_policy
+from wagtail.admin.forms.collections import BaseCollectionMemberForm, collection_member_permission_formset_factory
+
+from django.forms.models import modelform_factory
+
+
+class BaseMediaForm(BaseCollectionMemberForm):
+    class Meta:
+        widgets = {
+            "date": widgets.AdminDateInput,
+            "tags": widgets.AdminTagWidget,
+            "file": forms.FileInput,
+        }
+
+    permission_policy = media_permission_policy
+
+    def __init__(self, *args, **kwargs):
+        super(BaseMediaForm, self).__init__(*args, **kwargs)
+
+        # Remove unused fields.
+        for name in ("width", "height", "thumbnail"):
+            # these fields might be editable=False so verify before accessing
+            if name in self.fields:
+                del self.fields[name]
+
+    def clean_file(self):
+        error_messages = {
+            "mime_type": "%(filename)s (%(mime_type)s) is not an allowed %(type)s file.",
+        }
+
+        audio_types = [
```

Comment:
I can see us needing to update this list. Could you promote these to be class attributes or [defined in the module](https://github.com/wagtail/wagtail/blob/571b9e1918861bcbe64a9005a4dc65f3a1fe7a15/wagtail/images/fields.py#L12) for visibility and rename to convey their purpose (valid/allowed audio mime types)